utils/src/main/java/software/amazon/awssdk/utils/internal/MappingSubscriber.java

```diff
@@ -69,7 +69,7 @@ public void onNext(T t) {
         if (!isCancelled) {
             try {
                 delegateSubscriber.onNext(mapFunction.apply(t));
-            } catch (RuntimeException e) {
+            } catch (Exception e) {
```

Comment:
Should we catch Throwable?

Reply:
Why not? I don't think we can catch _too many_ exceptions here.

Reply:
@polyatail can you update the PR and add Throwable?

Reply:
Done.